public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -410,11 +411,12 @@ export class PrometheusDatasource extends DataSourceApi<PromQuery, PromOptions>
       end: 0,
     };
     const range = Math.ceil(end - start);
-
+    // target.stepOption specifies whether to use min, max or exact step
+    const stepOption = target.stepOption || 'min';
```

Comment:
Wondering if we couldn't use here the DEFAULT_STEP instead of `'min'`. And also, we could maybe use that in our UI component so component and logic can't unsync - that defaults at both places are always the same.

Reply:
Yeah that would be neat! Could we import DEFAULT_STEP from `PromQueryEditor.tsx`?

Reply:
Yes, that's what I was thinking as well.

Reply:
Awesome! I've changed it 👍